sycl/source/detail/scheduler/commands.cpp

```diff
@@ -76,6 +76,33 @@ void emitInstrumentationGeneral(uint32_t StreamID, uint64_t InstanceID,
 }
 #endif
 
+RTDeviceBinaryImage *
+retrieveAMDGCNOrNVPTXKernelBinary(const DeviceImplPtr DeviceImpl,
```

Comment:
This seems to partially overlap with [`retrieveKernelBinary`](https://github.com/intel/llvm/blob/sycl/sycl/source/detail/jit_compiler.cpp#L110) in `jit_compiler.cpp`. Is there any opportunity to refactor that?

Reply:
Yeap, I have rehomed it to [helpers.cpp](https://github.com/jchlanda/llvm/blob/jakub/jit_spec_const/sycl/include/sycl/detail/helpers.hpp#L260) and re-factored it to work with both use cases.